src/main/java/frc/robot/subsystems/DrivebaseSubsystem.java

```diff
@@ -174,19 +289,22 @@ private void drivePeriodic() {
     }
   }
 
-  // called in defense mode
+  @SuppressWarnings("java:S1121")
   private void defensePeriodic() {
-    // we want alternating pos and negative 45 degree angles
     int angle = 45;
     for (SwerveModule module : swerveModules) {
-      // the *= -1 operation multiplies the current variable by -1, stores it, and also returns the
-      // value. We can use this to alternate between 45 and -45 for each module.
+      // the *= -1 operation multiplies the current variable by -1, stores it, and also returns
+      // the value. We can use this to alternate between 45 and -45 for each module.
       module.set(0, angle *= -1);
     }
   }
 
-  @Override
-  public void periodic() {
+  /**
+   * Based on the current Mode of the drivebase, write the relevant periodic method.
```

Comment:
would this be run/call, not write? I guess write to the modules? Some methods do more then *just* write.